src/schema/index.js

```diff
@@ -141,6 +142,7 @@ type Subscription {
 
 input Options {
   storageBucket: String
+  stateful: Boolean
```

Comment:
I would like you to implement this a bit differently. The `storageBucket` is an option. Not sending the `storageBucket` option means persistence is off. There's no need for the additional `stateful` option.